packages/mica_case_studies/mica_case_studies.0.1.0/opam

```diff
@@ -0,0 +1,45 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Mica case studies (for OCaml Workshop 2024 talk)"
```

Comment:
Is this package expected to be of value for end-users? If it is only/primarily meant as an example or for test cases, please remove it from the publication, as per our policy that [Packages should provide some sort of utility for users](https://github.com/ocaml/opam-repository/wiki/Policies#3-packages-should-provide-some-sort-of-utility-for-users). However, if it is of some general utility, please add a description to explain.

Reply:
Ah I see, thanks for the heads-up! I've removed this package (it's now been isolated in a [separate repo](https://github.com/ngernest/mica_case_studies)) and its associated `.opam` file.